04_Scheduling/02_Scheduler.md

```diff
@@ -221,6 +221,11 @@ void idle_main(void* arg) {
 
 The idle task is scheduled a little differently: it should only run when there is nothing else to run. You wouldn't want it to run when there is real work to do, because it's essentially throwing away a full quantum that could be used by another thread.
 
+## Troubleshooting
```

Comment:
since i want to use pandoc to generate a pdf version of the notes when it is done, it is not perfect in handling titles not separated by a new line, so make sure to leave an empty line after each title.

Reply:
alright, ill fix that asap